JS_2/task1.js

```diff
@@ -0,0 +1,35 @@
+// Создать переменную “age” и присвоить ей значение 10
+
+const age = 10;
+
+//  Создать переменную “age_2” и присвоить ей значение 18
+
+const age_2 = 18;
+
+// Создать переменную “age_3” и присвоить ей значение 60
+
+const age_3 = 60;
+
+// Создать if в котором будите проверять значение переменной age
+
+if(age === 10) {
```

Comment:
don't see any requirements for this row. Is it needes?

Reply:
maybe check type, and another condition else